iree/hal/cuda/cuda_headers.h

```diff
@@ -8,5 +8,6 @@
 #define IREE_HAL_CUDA_CUDA_HEADERS_H_
 
 #include "cuda.h"  // IWYU pragma: export
+#include "cuda_runtime.h"
```

Comment:
seems cudaMemcpyKind I used to declare cuMemcpyAsync in dynamic_symbol_tables.h is included in this header file

Reply:
you'll want to copy that into this header - cuda_runtime.h isn't available and it's not worth adding (as it can make it really easy to misuse linkage - we don't use the runtime, only the device APIs)